include/z5/metadata.hxx

```diff
@@ -111,7 +119,18 @@ namespace z5 {
             dtype = types::Datatypes::zarrToDtype().at(j["dtype"]);
             shape = types::ShapeType(j["shape"].begin(), j["shape"].end());
             chunkShape = types::ShapeType(j["chunks"].begin(), j["chunks"].end());
-            fillValue = static_cast<double>(j["fill_value"]);
+
+            const auto fill_value_string = static_cast<std::string>(j["fill_value"]);
```

Comment:
This line causes the test failures, because `j["fill_value"]` cannot be cast to string if it's a double.